### PR TITLE
Added Huawei UPS2000 support

### LINKED ISSUE #10212: Huawei UPS 2000
**Please do not post bugs or ask questions**

**For help and support you can use our [Discord server](https://t.libren.ms/discord) or our [community site](https://community.librenms.org).**

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x ] Please provide **ALL** info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).

[discovery.txt](https://github.com/librenms/librenms/files/3171627/discovery.txt)
[poller.txt](https://github.com/librenms/librenms/files/3171628/poller.txt)
[snmpwalk.txt](https://github.com/librenms/librenms/files/3171630/snmpwalk.txt)

because ./pbin.sh doesnt work in freebsd !!!

- [x] Please provide as much detail as possible.

no graphs info in librenms :
UPS 2000
SNMP version: V100R002C02B150
Website: http://www.huawei.com
Email: support@huawei.com

- [x ] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Love librenms? Please consider supporting our collective:
👉  https://t.libren.ms/donations

### PR DESCRIPTION
**Pull request by PipoCanaja**
Fixes #10212

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
ready for review

File: includes/definitions/huaweiups.yaml [hunk 1]
```diff
@@ -6,9 +6,10 @@ icon: huawei
 rfc1628_compat: true
 over:
     - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_load, text: Load }
 discovery:
     -
-      sysDescr: Linux GSE200M
       snmpget:
```
Comment (murrant) [inline includes/definitions/huaweiups.yaml]: @PipoCanaja we try to reduce the number of snmpgets in os detection by having a qualifier before the snmpget.

Comment (PipoCanaja) [reply includes/definitions/huaweiups.yaml]: OK I'll add a list for sysDescr match. Cause Huawei does not put something constant in sysDescr for all their UPSes, instead they are quite clean in UPS-MIB.

**Comment by PipoCanaja** [comment]
Hi @murrant If you come around here, this one should be OK for merge.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-59-release-changelog-december-2019/10558/1

**Pull request merged**